es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = number + street;
+
+    console.log(`LearningFuze: ${address}`);
```

Comment:
Please make sure you're A) converting all concatenations, and B) not touching other parts of the code.

---

![image](https://user-images.githubusercontent.com/16143771/127082578-38b59641-b641-434e-93aa-219a586f5051.png)

---

![image](https://user-images.githubusercontent.com/16143771/127082521-11212a3a-17a4-45ab-9e49-024bac9d30ea.png)